docs/index.md

```diff
@@ -281,7 +281,7 @@ This data store keeps values added to it during the life cycle of the specificat
 import {DataStoreFactory, DataStore} from 'gauge-ts';
 
 // Adding value
-let scenarioScore: DataStore = DataStoreFactory.getScenarioDataStore();
+let specStore: DataStore = DataStoreFactory.getSpecDataStore();
 scenarioScore.put("element-id", "455678");
```

Comment:
`scenarioScore.put("element-id", "455678");` should be `specStore.put("element-id", "455678");`